obsproject/obs-studio
# obs-ffmpeg: Add pause visibility behaviour

### Description


Based on https://github.com/obsproject/obs-studio/pull/6112

Add ability to pause local media src when source is not visible, and unpause when not visible
Use combobox like slideshow/vlc source visibility behaviour

<img width="905" alt="Visibility" src="https://user-images.githubusercontent.com/33432858/179237597-6f7f5ef5-3541-4b64-add0-2c9fa478e33e.png">

Use a similar UI code than PR https://github.com/obsproject/obs-studio/pull/6239

### Motivation and Context



Let user choose to pause a media when not visible
Another step to align media source features with other source.

### How Has This Been Tested?



Mac os Intel 11.6.7
- Check that the two mode of Restart playback..., is keep from a previous version of Obs
- Check that modify checkbox "Restart playback...) (in not local file mode), 
or new combobox "Visibility Behaviour" (in local file mode), correctly update UI
- Check that the three behaviour works as expected with a video file (even in loop mode)

### Types of changes


- New feature (non-breaking change which adds functionality)






### Checklist:
